### ISSUE TITLE
Добавить возможность перемещать/копировать wpt между треками.

### ISSUE THREAD

**Issue by judgerain** [wladich/nakarte#661]
В данный момент, к примеру, если хочется разбить трек на несколько кусков, wpt приходится пересоздавать вручную. 
Хорошо бы добавить в контекстное меню пункты: "copy to track"  и "move to track" с последующим выбором трека в который копировать/перемещать.

**Comment by wladich** [comment]
Да, идея хорошая. Мне, правда, чаще нужно бывает переместить все точки, когда вдруг понял, что создал их не в том треке.

**Comment by slazav** [comment]
А, кстати, мне из этой серии очень не хватало операции "перенести сегмент в другой трек". Я пытался применять для этой цели join, но тогда кусок трека дублируется (это, кстати, мне кажется странным поведением), и его надо удалять, и соединительную линию.

**Comment by myadzel** [comment]
Предложу более универсальный вариант: экспорт и импорт точек из трека.

**Comment by sikmir** [comment]
Да, возможности копировать wpt очень не хватает, например есть пачка треков с точками из разных отчетов, точек очень много (>100), хочу копировать только интересующие меня точки в свой трек, и вот тут начинается боль и страдание)
